sandbox/cgroup/tracking.go

```diff
@@ -0,0 +1,243 @@
+package cgroup
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/godbus/dbus"
+
+	"github.com/snapcore/snapd/dbusutil"
+	"github.com/snapcore/snapd/features"
+	"github.com/snapcore/snapd/logger"
+)
+
+var osGetuid = os.Getuid
+var osGetpid = os.Getpid
+var cgroupProcessPathInTrackingCgroup = ProcessPathInTrackingCgroup
+
+var ErrCannotTrackProcess = errors.New("cannot track application process")
+
+func CreateTransientScope(securityTag string) error {
```

Comment:
this needs a doc comment, also should be called something like CreateTransientScopeFor(App?)Tracking to be consistent with ConfirmSystemdServiceTracking